src/ui/ObjectControls.js

```diff
@@ -785,14 +787,16 @@ ObjectControls.prototype.translatePivotByMouse = function () {
 // Translate in WorldCS, translation is scaled with root scale matrix
 ObjectControls.prototype.translatePivotInWorld = function (x, y, z) {
   const pos = this.objectPivot.position;
+  const oldPos = this.objectPivot.position.clone();
   pos.applyMatrix4(this.object.matrixWorld);
   pos.setX(pos.x + x);
   pos.setY(pos.y + y);
   pos.setZ(pos.z + z);
   const invWorldMat = new THREE.Matrix4().getInverse(this.object.matrixWorld);
   pos.applyMatrix4(invWorldMat);
 
-  this.dispatchEvent({ type: 'change' });
+  const shift = oldPos - pos;
```

Comment:
Does difference operator exist for the Vector3?

Reply:
Yes, thank you
I fix it